tests/formatters/json.js

```diff
@@ -0,0 +1,33 @@
+(function(){
+    "use strict";
+    var Assert = YUITest.Assert;
+
+    YUITest.TestRunner.add(new YUITest.TestCase({
+
+        name: "JSON formatter",
+
+        "File with no problems should say so": function() {
+            var result = { messages: [], stats: [] },
+                actual = CSSLint.getFormatter("json").formatResults(result, "path/to/FILE", {fullPath: "/absolute/path/to/FILE"});
+            Assert.areEqual("{\"messages\":[],\"stats\":[]}", actual);
+        },
+
+        "Should have no output when quiet option is specified and no errors": function() {
```

Comment:
I'm not sure I understand what you are asking here.

The way `JSON.stringify` works means that the result object being passed in there would still show `{"messages":[],"stats":[]}` if quiet is not true.

Formatters have nothing to do with parsing the options, if `csslint` isn't sending in a properly valued `quiet` option that's a bug in the rest of `csslint`, right?